src/estimators.jl

```diff
@@ -701,9 +701,9 @@ function LGBMRanking(;
     max_cat_threshold = 32,
     cat_l2 = 10.0,
     cat_smooth = 10.0,
-    metric = [""],
+    metric = ["ndcg"],
```

Comment:
Updated this default as the default objective for this estimator is lambdarank and these two usually go together. Unlike for binary, multiclass or regression objectives, where the empty "" metric is filled with default value corresponding to the pre-defined objective, the empty string doesn't assign the metric in this case.